code/src/components/AddNote.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { notereminder } from 'reducers/reminders';
+import { AddBtn, CounterP } from './Styling/GlobalStyle';
+
+export const AddNote = () => {
+  const [inputValue, setInputValue] = useState('');
+  const [charCount, setCharCount] = useState(0);
+  const dispatch = useDispatch();
+
+  const capitalizeFirstLetter = (str) => {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+  }
+
+  const onInputChange = (event) => {
+    setInputValue(event.target.value);
+    setCharCount(event.target.value.length);
+  }
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+    const newNote = {
+      id: Date.now().toString(),
```

Comment:
This id works for this project, but when building larger applications with multiple users there's a risk that two posts get the same id if posted at the same time. There are id-generators like uuid (npm package) that prevents this if you want to have look.